labelbox/orm/model.py

```diff
@@ -91,8 +102,12 @@ def Enum(enum_cls: type, *args):
     def Json(*args):
         return Field(Field.Type.Json, *args)
 
+    @staticmethod
+    def List(list_cls: type, *args):
```

Comment:
noticing you modeled `List` similar to `Enum`

jw, but what is the reason why `Enum` and `List` (your new static method) are separate from the `Field.Type` enums?

Reply:
The List takes in a data type, whereas Field.Types can't do that. The new data type defines what kind of data is stored in the List, so it's mostly done to provide clarity to the reader. Currently, we're not doing much with that data type, but it'll be good to have this structure for transparency, and we may come back to do something with it in the future